search-app/src/cmr/search/site/data.clj

```diff
@@ -0,0 +1,40 @@
+(ns cmr.search.site.data
+  "The functions of this namespace are specifically responsible for generating
+  data structures to be consumed by site page templates.
+
+  Of special note: this namespace and its sibling `page` namespace are only
+  ever meant to be used in the `cmr.search.site` namespace, particularly in
+  support of creating site routes for access in a browser.
+
+  Under no circumstances should `cmr.search.site.data` be accessed from outside
+  this context; the data functions defined herein are specifically for use
+  in page templates, structured explicitly for their needs.")
+
+(defn get-index
+  "Return the data for the index page (none for now)."
+  [request]
+  {})
+
+(defn get-landing-links
```

Comment:
I don't understand the difference between the landing pages link and the EOSDIS landing pages link. I believe we are only supposed to create links for EOSDIS landing pages, so those would be the only type. Within the EOSDIS landing pages there will be two different types of links:

1. Collections which have a DOI which we will link to
2. Collections which do not have a DOI, which we will return a link to the CMR HTML collection page.